src/electron/go_vpn_tunnel.ts

```diff
@@ -199,9 +188,11 @@ export class GoVpnTunnel implements VpnTunnel {
       // connected.
       console.error(`could not stop routing: ${e.message}`);
     }
+    this.resolveAllHelpersStopped();
```

Comment:
Shouldn't we communicate to the user if disconnection fails?
Perhaps we should reject the promise in case of failure?

Reply:
Good point. One operation shouldn't fail the call, but we can keep track of the errors and throw if both fail.